.github/workflows/zephyr.yml

```diff
@@ -39,6 +39,24 @@ jobs:
           west init -l
           west update --fetch-opt=--filter=tree:0
 
+      # Temporary check until we change west.yml and stop nesting rimage
+      # and tomlc99 inside sof which will cleanly separate them from
+      # XTOS submodules and... temporarily break every CI, which is why
+      # it hasn't been done yet.
+      - name: git submodules consistency
+        run: |
+
+          cd workspace/sof
+          git submodule update --init --recursive
+
+          west update
+          if git status --porcelain=v2 | grep ^ ; then
+             git status
+             echo 'FAIL: inconsistency between git submodules and west.yml!'
+             echo 'Always use "git status"'
```

Comment:
What are you telling people to do here ? By itself ```git status``` wont fix the problem.
Btw, why not just convert rimage to west and avoid temp W/A ?

Reply:
> What are you telling people to do here ? By itself git status wont fix the problem.

The error message says "inconsistency between git submodules and west.yml" and the check name is "git submodule consistency". There's also a comment in west.yml above the line that people changed.

> Btw, why not just convert rimage to west and avoid temp W/A ?

That's explained in the comment above. rimage is already west, the longer-term fix is to change its location so it does coincide with the XTOS submodule. However this will (temporarily) break CI as every rimage change as we just saw in https://github.com/zephyrproject-rtos/zephyr/pull/54700 and https://github.com/zephyrproject-rtos/zephyr/pull/56099

Reply:
@lgirdwood I added a direct reference to the detailed rimage submodule instructions in sof/west.yml, I should have done that from the start.